flood_api/utils/validation_helpers.py

```diff
@@ -5,6 +5,7 @@
 from flood_api.settings import settings
 
 GLOFAS_ROI = settings.glofas_roi
+INVALID_STATUS_CODE = 404
```

Comment:
404 does not equal invalid. It should not be our default for invalid requests. I am not sure if it's neccessary to store 404 in a constant?